Analyzer/test/Makefile.in

```diff
@@ -40,7 +40,7 @@ LIBSTOPTAGGER += -L$(TTTDIR) -lTopTagger
 INCLUDESDIRS  += $(shell root-config --cflags)
 
 # ROOT libraries
-LIBS          += $(shell root-config --glibs)
+LIBS          += $(shell root-config --glibs) -lGenVector
```

Comment:
did you need to add `-lGenVector`? I thought it was needed but I think it is depreciated from like ROOT 5.

Reply:
yeah, early on in these changes, I could not get things to compile (undefined reference when I wanted to include for the ROOT::Math::Boost method, I believe)